notebooks/examples/configure_check_conditions.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>"Now we know what the return value looks like."</p><p> </p><p>*Capslock on all new sentences</p><p> </p><p>" that validates that the ratio between the train and ..."</p><p> </p><p>"check which accepts a condition name and a function"</p><p> </p><p>"and should return either a boolean or a <code>ConditionResult</code> containing a boolean and optional extra info that will be displayed in the Conditions Summary table"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>